src/components/text-block.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC, ReactNode } from 'react';
+
+export type TextBlockProps = {
+  title: string;
+  link?: { label: string; href: string };
+  number?: number;
+  unit?: string;
+  highlightNumber?: boolean;
+  children: ReactNode;
+};
+
+export const TextBlock: FC<TextBlockProps> = ({ title, children, link, number, unit, highlightNumber = false }) => (
+  <div>
+    {number && (
+      <p className={`text-xl lg:text-xxl font-bold ${highlightNumber ? 'fancy-text' : ''}`}>
```

Comment:
This is a copy of the guetzli component right? Do you even need such a component in your project?
And if you do you need to make sure everything works as expected. currently the number highlighting wouldn't work since you haven't defined the css class `fancy-text`.